kernel/collection/array/array.c

```diff
@@ -0,0 +1,104 @@
+//*****************************************************************
+// This file is part of CosmOS                                    *
+// Copyright (C) 2020 Tom Everett                                 *
+// Released under the stated terms in the file LICENSE            *
+// See the file "LICENSE" in the source distribution for details  *
+// ****************************************************************
+
+#include <collection/array/array.h>
+
+#include <mm/mm.h>
+#include <panic/panic.h>
+#include <console/console.h>
+
+struct array* arrayNew(uint32_t size) {
```

Comment:
Again, another style comment, but for consistency's sake I think we should standardize on a symbol name convention, especially for symbols that are intended to be visible outside the module they're defined in.  Personal preference and an exercise of personal privilege here, but since I already established the practice of using underscores rather than camelcase, that's what I'm going to run with going forward.

Reply:
I'll fix them.  I have no issue with underscores.   My brain was in Java mode when I wrote this :)

Reply:
lotsa fixes added.